### DISCUSSION TITLE
What's Next? A Bit Lost.... Controlling Fermentation Temperature

### DISCUSSION THREAD

**Discussion by phillc73** [BernhardSchlegel/BierBot-Bricks#7] [Q&A] [answered]
I've built a BierBot Brick! Happy days. I've plugged it into the mains power and nothing went bang, there's no smoke or smell of burning plastic. I count that as a success already.

I've flashed the firmware and the Brick shows up as active in the Bierbot Bricks website.

I've added some equipment. In this case, I want to use my Brick to control the heating for a fermenter. Fermenter is in a cellar but I want it at 22 degrees. I've dragged the relay and temperature sensor from the Brick across to the fermenter.

Essentially now I'm stuck! How do I set the target temperature for the probe, so that when the temperature drops below the target, the heating element kicks in? 

I don't need to setup any recipe, as I manage all of that elsewhere. I tried going into "Manual", but it says I still need to setup a device.

Not sure where to go from here?

**Comment by BernhardSchlegel** [comment, marked as answer]
Hey @phillc73 , great work - your almost there! And congrats on not burning your house down. And cool that you are aboard! :) Currently, we have 2 types of hardware:

- a mashtun / biab: which requires at least a temperature sensor and heating being assigned.
- a fermenter: which requires at least a temperature sensor and cooling being assigned.

Once the minimum requirements are met, you can use the device in manual or for brewing recipes in automatic. If it's not ready, you can click on the top right warning icon of the device to get a hint what's missing.

Which one did you pick? Did you assign the relay/switch? Did you save your device? Do you need a generic "fermentation heating device?"

Can you share a screenshot of your equipment screen?

I'm currently working on a documentation, device types will be explained there. Thanks for reaching out!

Cheers,
Bernhard.

**Reply by phillc73** [reply]
Thanks @BernhardSchlegel 

I think the problem here is that I haven't assigned any cooling to the fermenter. This is because I only need heating for the fermenter. It's located in a cool location, but I want ale yeast fermenting at 22 deg C.

![image](https://user-images.githubusercontent.com/815126/122631479-5bfac900-d0cc-11eb-9d72-5e214e2fa960.png)

Perhaps your suggestion of a generic fermentation heating device is what I need?

**Reply by BernhardSchlegel** [reply]
Yep! You're absolutely right. I'll add it to the backlog. Expect it next week. Use can use a mashtun in the meantime as workaround. You'll need to add your temperature steps for automatic mode to the mashsteps for it to work, however. Manual mode should work just fine.

**Reply by phillc73** [reply]
Great! Everything works in manual mode when pretending my fermenter is a mash tun!

![image](https://user-images.githubusercontent.com/815126/122651097-eb3cc680-d136-11eb-967e-5b55f64495c0.png)

Really appreciate the fermenter heating device coming next week. However, for our brew tomorrow, I'm pretty sure we will use this, and are clever enough not to confuse the fermenter with the mash tun.

Love your work here and would be keen to chat about future plans and features.

**Reply by BernhardSchlegel** [reply]
Glad to hear! Thanks for the kind words!

I'm happy to chat about future features, just shoot me a message hello [a d] BierBot.com
